arch/mips/boot/compressed/dbg.c

```diff
@@ -646,7 +646,7 @@ static void set_res(void)
 	 * syscnt 0x0
 	 */
 
-	const u64 smode1_ = 0x13422004b1;
+	const u64 smode1_ = 0x740806504;
```

Comment:
So we have `gcont rgbyc` here, but override it to `gcont ycrcb` on line 653. Sure, that’ll probably be best for most people with current equipment (such as HDMI adapters).

Reply:
I realized that nowadays gcont ycbcr has wider use than rgb. Ycbcr isnt supported only in composite wires. Hdmi, scart, d-terminal, component, s-video, vga: all of them supports Ycbcr. But, yes, its true that I forgot that I dumped regs after gcont rgb applied, so maybe your values will be more accurate. Feel free to close this pr and apply your values.
This pr for get rid of 1080p, cause 1080p isnt supported by some ps2 hardware. PAL signal is more standart (even with ycbcr).
I choose PAL not NTSC just because it has larger height, so more terminal lines fit the window. However, now I think that for printk it will be enough NTSC signal, cause printed lines count is limited and will fit the screen anyway. Also NTSC is more popular.